can/parser_pyx.pyx

```diff
@@ -15,8 +15,6 @@ import os
 import numbers
 from collections import defaultdict
 
-cdef int CAN_INVALID_CNT = 5
-
```

Comment:
Need to figure out why this was added and if we can remove it, this is a threshold on the parser's valid which is already using a threshold. This should be in the C++ code anyway.